tools/configure/configure.ml

```diff
@@ -271,13 +271,15 @@ let default = {
 }
 
 let devel state = { state with
-  local = true;
   bin_annot = true;
   annot = true;
   warn_error = true;
   dune_profile = "dev";
+  interactive = false;
+  output_summary = true;
+  prefix = Some (Filename.concat (Sys.getcwd ()) "_build_vo/default");
 }
-let devel_doc = "-local -annot -bin-annot -warn-error yes"
+let devel_doc = "-annot -bin-annot -warn-error yes"
```

Comment:
With the recent changes, the `devel` profile is not equivalent anymore to a fixed set of flags. You could expand this doc entry with `-prefix $(pwd)/_build_vo/default` but there would still be the issue of the Dune profile which is set to `dev` and doesn't have a dedicated option. So it is the entire profile doc that should be reworked (now or in your subsequent configure refactoring PR).